tests/test_executable.py

```diff
@@ -15,6 +15,7 @@
 from StringIO import StringIO
 from io import BytesIO
 import os
+import logging
```

Comment:
Can we keep the import order here? Oh wait, there's the `StringIO` import at the start, so nevermind, there's no order :D

Reply:
I find this obsession with import ordering strange. It's supposed to make it easier to find unused imports or something?  Any decent IDE does that automatically. I'd just separate custom modules, standard libraries and pypi packages.